File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int mul = 1;
        int add = 0;
        int sqr=0;
        for(int i=0; i<num_list.length; i++){
            mul *= num_list[i];
            add += num_list[i];
        }
        sqr = (int)Math.pow(add,2);
        if(sqr > mul) answer = 1;
        else answer =0; 
        return answer;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    var answer = 0;
    let mul =1; 
    let pow = 0;
    for(let x of num_list){
        mul *=x;
        pow+= x;
    }
    if(mul <pow**2){
        answer = 1;
    }else answer=0;
    return answer;
}